test/libsolidity/semanticTests/abiEncoderV1/memory_params_in_external_function.sol

```diff
@@ -0,0 +1,18 @@
+contract C {
+    function f(bytes memory a, bytes calldata b, uint[] memory c)
```

Comment:
I think the really cool thing here is that you can mix memory and calldata.

Reply:
I'm still not sure this will be *that* useful *that* often, but it sure doesn't hurt :-).